test/e2e/kfto_kueue_sft_test.go

```diff
@@ -0,0 +1,125 @@
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	. "github.com/project-codeflare/codeflare-common/support"
+
+)
+
+func TestPytorchjobWithSFTtrainer(t *testing.T) {
+	test := With(t)
+	test.T().Parallel()
+
+	// Create a namespace
+	namespace := test.NewTestNamespace()
+	config := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "my-config",
+			Namespace: namespace.Name,
+		},
+		BinaryData: map[string][]byte{
+			"config.json":                   ReadFile(test, "config.json"),
+			"twitter_complaints_small.json": ReadFile(test, "twitter_complaints_small.json"),
+		},
+		Immutable: Ptr(true),
+	}
+
+	config, err := test.Client().Core().CoreV1().ConfigMaps(namespace.Name).Create(test.Ctx(), config, metav1.CreateOptions{})
+	time.Sleep(20 * time.Second)
+	test.Expect(err).NotTo(HaveOccurred())
+	test.T().Logf("Created ConfigMap %s/%s successfully", config.Namespace, config.Name)
+
+	tuningjob := &kftov1.PyTorchJob{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "PyTorchJob",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kfto-sft",
+			Namespace: namespace.Name,
+		},
+		Spec: kftov1.PyTorchJobSpec{
+			PyTorchReplicaSpecs: map[kftov1.ReplicaType]*kftov1.ReplicaSpec {
+				"Master": &kftov1.ReplicaSpec{
+					Replicas: 1,
+					RestartPolicy: "Never",
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							Containers: []corev1.Container{
+								{
+									Name: "pytorch",
+									Image: "quay.io/tedchang/sft-trainer:dev",
+									ImagePullPolicy: corev1.PullIfNotPresent,
+									Command: []string{"python", "/app/launch_training.py"},
+									Env: []corev1.EnvVar{
+										{
+											Name: "SFT_TRAINER_CONFIG_JSON_PATH",
+											Value: "/etc/config/config.json",
+										},
+									},
+									VolumeMounts: []corev1.VolumeMount{
+										{
+											Name: "config-volume",
+											MountPath: "/etc/config",
+										},
+									},
+								},
+							},
+							Volumes: []corev1.Volume{
+								{
+									Name: "config-volume",
+									VolumeSource: corev1.VolumeSource{
+										ConfigMap: &corev1.ConfigMapVolumeSource{
+											LocalObjectReference: corev1.LocalObjectReference{
+												Name: "my-config",
+											},
+											Items: []corev1.KeyToPath{
+												{
+													Key: "config.json",
+													Path: "config.json",
+												},
+												{
+													Key: "twitter_complaints_small.json",
+													Path: "twitter_complaints_small.json",
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				}
+			},
+		},
+	}
+	// Need to implement the test client for the pytorchJob
+	// tuningjob, err := test.Client().kvtov1().pytorchJob().create() 
```

Comment:
@z103cb @astefanutti I am trying to wrap my head around how this is implemented..

Reply:
@tedhtchang this should be added into the test client. I've created https://github.com/project-codeflare/codeflare-common/pull/38. Once it's merged you should be able to bump the codeflare-common version and uncomment this.

Reply:
@astefanutti Thanks! I was able to create a pytorchjob.